src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -226,11 +233,62 @@ class PeerActor(
     context become waitingForConnectionResult(newConnection, noRetries)
   }
 
+  def handleEvmMptFastDownload(rlpxConnection: RLPxConnection): Receive = {
```

Comment:
@adamsmo i'm wondering...should this code be performed by one (or many) separate actors that would be in charge of retrieving info for other nodes? This is an open question in order at least to think about this.

As pros to do so I can think of:

- Avoid accesing the storages within the main Peer actor (this can be done within a Future for example)
- Encapsulating the functionality
- To be able to use multiple actor for this (as it's done in FastSyncController with Handlers). This is not a concrete need though

On the other side, we might be adding more complexity that might not be needed.

ps: this is also related to https://github.com/input-output-hk/etc-client/pull/50

summoning @LukasGasior1